armsrc/appmain.c

```diff
@@ -1077,7 +1070,6 @@ void UsbPacketReceived(uint8_t *packet, int len) {
 #endif
 		case CMD_BUFF_CLEAR:
 			BigBuf_Clear();
-			BigBuf_free();
```

Comment:
it shall empty big buff.

Reply:
Merge conflict with old version from iceman fork.